### PR TITLE
Portfolio: validate contact form with JavaScript

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

[Portfolio: validate contact form](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md)

### Requirements 🚀

We have reviewed the following on the page for best accessibility : 

For this project: 
- [x] The content of the email field has to be in lower case.
- [x] When the user submits the form, you check if the email is in lower case.
- [x] If the validation is OK, the form is sent.
- [x] If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.
- [x] When you create the error message, you should stick to the design guidelines

### Pair Programming 🎳

For this exercise, PP was required, we worked very closely with @fabianofrank to make the best code possible for the assignment.

### Additional requirements

- [ ] When the form validation is not OK, you can provide the user with their text converted to lower case, so they can use it directly in the input field.

Happy Review :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,20 +113,46 @@ <h3 class="knowledges">Skills</h3>
...
-              <button class="button form-button" type="submit">Get in Touch</button>
-          </ul>
+            <div class="form-field">
+              <label for="Name"></label>
+              <input
+                type="name"
+                maxlength="30"
+                id="name"
+                name="user_name"
+                placeholder="Full name">
```
Comment (dasileker) [inline index.html:128]: - [ ] The name field also should be required.

File: index.html [hunk 2]
```diff
@@ -113,20 +113,46 @@ <h3 class="knowledges">Skills</h3>
...
+            </div>
+            <div class="">
+              <label for="message"></label>
+              <textarea
+                type="text"
+                id="msg"
+                maxlength="500"
+                name="user_message"
+                placeholder="Write your message here">
+              </textarea>
```
Comment (dasileker) [inline index.html:150]: - [ ] also the textarea should be required no one want an empty message.

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @benjp009,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benjp009** [comment]
Hi @dasileker
Thanks so much for your review, yes indeed you are right, I had removed them to make it easier for testing but thanks for reminding me to put them back on required. 

I have made the necessary changes

**Review by dasileker — APPROVED** [review]
## **Hi @benjp009 ,**


## **Congratulations! 🎉 Project Approved**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benjp009** [comment]
@dasileker Thanks for this fast review

**Pull request merged**